content/docs/ui/sending-email/getting-started-with-automation.md

```diff
@@ -168,3 +168,16 @@ _To delete an automation:_
 1. On the automation dashboard, find the automation you want to delete.
 1. Hover over the action menu and select the delete icon.
 1. To delete the automation, click **Confirm**.
+
+## Automation Use Cases and Examples
+
+### Segment-triggered Automation
```

Comment:
I think this would be "Segment-triggered nurture series" 

The use case lead-in could be "A valuable use case for Automation is to nurture your free customers to become paid using a drip series. The series will send automatically to free users until they reach the end of the email sequence or they upgrade to paid, whichever comes first. Here's how:"

Reply:
This is great. Thank you!